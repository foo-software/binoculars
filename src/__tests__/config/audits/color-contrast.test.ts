import auditColorContrast from '../../../config/audits/color-contrast';

describe('color-contrast audit', () => {
  it('should match snapshot', () => {
    expect(auditColorContrast('en')).toMatchSnapshot();
  });
});
