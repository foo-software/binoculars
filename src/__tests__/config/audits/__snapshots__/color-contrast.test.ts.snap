// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`color-contrast audit should match snapshot 1`] = `
Object {
  "description": "Low-contrast text is difficult or impossible for many users to read. [Learn more](https://web.dev/color-contrast/).",
  "title": "Background and foreground colors have a sufficient contrast ratio",
}
`;
